debian/control

```diff
@@ -31,42 +31,43 @@ Description: Astrobee flight software configuration.
 Package: astrobee-comms
 Section: comm
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}
```

Comment:
maybe should leave this in even if it's empty, in the hopes we may fix shlibdeps in the future?

Reply:
I took it out because lintian complained about it. Maybe we can put it back once we make it work? Although, I'm not sure if it will ever work with xenial/focal and our chroot setup. dpkg has a new feature that can help us but it's not available for focal or xenial.